Ark.Tools.ResourceWatcher/IResourceInfo.cs

```diff
@@ -14,11 +14,16 @@ public interface IResourceMetadata
         /// <summary>
         /// The "version" of the resource. Used to avoid retrival of the resource in case if the same version is already been processed successfully
         /// </summary>
-        LocalDateTime? Modified { get; }
+        LocalDateTime Modified { get; }
         /// <summary>
-        /// The "versions" of the resource. Used to manage multiple sources for the resource to avoid retrival of the resource in case if the last version is already been processed successfully
+        /// The "versions" of the resource. Used to manage multiple sources for the resource.
+        /// The resource will be processed when a new source will be add or at least one source have an updated modified.
         /// </summary>
+#if (NET472 || NETSTANDARD2_0)
         Dictionary<string, LocalDateTime> ModifiedMultiple { get; }

```

Comment:
I do like the concept of 'Source' that you used in the comment above. How about renaming this to ModifiedSources ?